scripts/bitops-config/README.md

```diff
@@ -47,6 +47,12 @@ nested:
   foo: true
 ```
 
+### Sample local scripts
+```
+export DEBUG=true; export DEEP_DEBUG=; export BITOPS_DIR=/Users/Phil/Desktop/BitOps/bitops; export SCRIPTS_DIR=$BITOPS_DIR/scripts; export SCHEMA_FILE=$SCRIPTS_DIR/helm/bitops.schema.yaml; export BITOPS_CONFIG_FILE=$BITOPS_DIR/bitops.config.yaml; /Users/Phil/Desktop/BitOps/bitops/scripts/bitops-config/convert-schema.sh $SCHEMA_FILE $BITOPS_CONFIG_FILE
```

Comment:
maybe remove `/Users/Phil/Desktop/Bitops` and do something like `/path/to/bitops/scripts/bitops-config/convert-schema` ?

even better would be to show how it works from a docker command like:
```
docker run --rm --name bitops-local \
  -e ENVIRONMENT="foo" \
  -e DEBUG="true" \
  -e DEEP_DEBUG="" \
  --entrypoint="/opt/bitops/scripts/bitops-config/convert-schema.sh" \
  -v /path/to/operations:/opt/bitops_deployment \
  bitops:latest \
  "/opt/bitops/scripts/example-tool/bitops.schema.yaml" \
  "/opt/bitops_deployment/foo/example-tool/bitops.config.yaml" 
```

Reply:
Got it. I remember why I wrote this in the first place. 
So, right under this there is a great example using the same docker container method you use here to test. 

I added the local sample (clumsily it appears) just to point out you could run it as a standalone script.